Bottom: 0ea170dc9ebd7eb7f5698295321f155f1cca217f
Top:    05d1c5cc048c15a4857528e0ff651d5f5f047951
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 03:44:27 +0000

Refresh of configure-ac-remove-cc_or_cxx_

---

diff --git a/configure.ac b/configure.ac
index 34eddb1..6694f57 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1169,15 +1169,12 @@ ENABLE_BUILD_WITH_CXX=no)
 case "$ENABLE_BUILD_WITH_CXX" in
   yes)
     CC_OR_CXX="$CXX"
-    CC_OR_CXX_FLAGS="$CXXFLAGS"
     ;;
   no)
     CC_OR_CXX="$CC"
-    CC_OR_CXX_FLAGS="$CFLAGS"
     ;;
 esac
 AC_SUBST(CC_OR_CXX)
-AC_SUBST(CC_OR_CXX_FLAGS)
 
 # font filename
 FONTFILENAME=${FONTFILENAME:-"default_font"}
